### PR TITLE
🐛 Minor fixes: resources files path, gateway openapi specs

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- 🐛 Fixes generation of opeanpi: how `/` can be evil :-)
- ignores `gateway/openapi.json` : can regenerate and "offical" version can be found in appmtion-exchange repo
- Adds example for Account Request form

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by GitHK — APPROVED** [review]

File: api/specs/web-server/openapi.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def main():
     openapi = create_openapi_specs(app, remove_main_sections=False)
 
     # .yaml
-    oas_path = webserver_resources.get_path("/api/v0/openapi.yaml")
+    oas_path = webserver_resources.get_path("api/v0/openapi.yaml").resolve()
```
Comment (bisgaard-itis) [inline api/specs/web-server/openapi.py:85]: 💩

File: packages/models-library/src/models_library/api_schemas_webserver/auth.py [hunk 2]
```diff
@@ -10,6 +10,28 @@
...
 
+    class Config:
+        # NOTE: this is just informative. The format of the form is defined
+        # currently in the front-end and it might change
+        # SEE image in  https://github.com/ITISFoundation/osparc-simcore/pull/5378
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "example": {
+                "form": {
+                    "firstName": "James",
+                    "lastName": "Maxwel",
```
Comment (bisgaard-itis) [inline packages/models-library/src/models_library/api_schemas_webserver/auth.py:21]: Man, everywhere I look in this repo I find this guy 🤔

**Review by bisgaard-itis — APPROVED** [review]
Thanks

File: packages/models-library/src/models_library/api_schemas_webserver/auth.py [hunk 3]
```diff
@@ -10,6 +10,28 @@
...
+                    "email": "maxwel@email.com",
+                    "phone": "+1 123456789",
+                    "company": "EM Com",
+                    "address": "Infinite Loop. California",
+                    "country": "USA",
+                    "application": "Antenna_Design",
+                    "description": "Description of sometin",
```
Comment (sanderegg) [inline packages/models-library/src/models_library/api_schemas_webserver/auth.py:28]:
```suggestion
                    "description": "Description of something",
```

**Review by sanderegg — APPROVED** [review]

File: packages/models-library/src/models_library/api_schemas_webserver/auth.py (hunk 2 above)
Comment (pcrespov) [reply packages/models-library/src/models_library/api_schemas_webserver/auth.py:21]: very important guy :-)

**Pull request merged**